### DISCUSSION TITLE
NTR: Medical/physician specialty role

### DISCUSSION THREAD

**Discussion by LCCarmody** [ufbmi/OMRSE#330] [General]
Medical Action ontology has a need for medical specialty evaluation (i.e., neurologist evaluation).

I have already created most of the terms needed based on the [ABMS](https://www.abms.org/member-boards/specialty-subspecialty-certificates/) board specialties, but we have yet to build logical terms.

  For MAxO, we are making the term medical specialty evaluation, but it would be great if another ontology had the specialty or the specialty role in an ontology where we could rely on their structure and maintenance of terms.  In early searches, we thought it might fit in OMRSE, but we weren't certain.

Let me know if you would consider adding terms under physician role (OMRSE:00000013)

https://github.com/monarch-initiative/MAxO/issues/155

Terms currently in MAxO which may be relevant. 

'psychiatrist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000714>
'pulmonologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000693>
'ophthalmologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000703>
'rheumatologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000694>
'endocrinologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000685>
'general surgeon evaluation' <http://purl.obolibrary.org/obo/MAXO_0000719>
'plastic surgeon evaluation' <http://purl.obolibrary.org/obo/MAXO_0000713>
'otolaryngologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000706>
'vascular surgeon evaluation' <http://purl.obolibrary.org/obo/MAXO_0000720>
'child neurologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000717>
'gastroenterologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000686>
'medical oncologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000691>
'medical specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000665>
'orthopedic surgeon evaluation' <http://purl.obolibrary.org/obo/MAXO_0000478>
'medical professional evaluation' <http://purl.obolibrary.org/obo/MAXO_0000483>
'dermatology specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000671>
'occupational therapist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000732>
'pediatric cardiologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000711>
'pain medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000677>
'anesthesiology specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000667>
'sleep medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000679>
'allied health professional evaluation' <http://purl.obolibrary.org/obo/MAXO_0000731>
'family medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000673>
'sports medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000678>
'nuclear medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000701>
'sports medicine orthopedist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000704>
'infectious disease internist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000690>
'internal medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000680>
'thoracic and cardiac surgeon evaluation' <http://purl.obolibrary.org/obo/MAXO_0000721>
'emergency medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000672>
'geriatric medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000675>
'adolescent medicine specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000681>
'clinical laboratory geneticist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000699>
'clinical biochemical geneticist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000697>
'neurological surgery specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000700>
'cardiovascular disease internist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000682>
'critical care medicine internist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000684>
'child and adolescent psychiatrist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000715>
'colon and rectal surgery specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000670>
'developmental-behavior pediatrician evaluation' <http://purl.obolibrary.org/obo/MAXO_0000709>
'pediatric anesthesiology specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000669>
'psychiatry and neurology specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000723>
'obstetrics and gynecology specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000702>
'clinical cardiac electrophysiologist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000683>
'clinical genetics and genomics specialist evaluation' <http://purl.obolibrary.org/obo/MAXO_0000698>

As I mentioned we have an allied health profession role request, but I will open a separate ticket for it.

Thanks.

**Comment by matentzn** [comment]
Is there any way we can make it easier to help you add these?

**Comment by hoganwr** [comment]
We have our monthly meeting on Wednesday. I can push this up on the agenda.

**Comment by LCCarmody** [comment]
I will try to be there. Thanks.

**Comment by hoganwr** [comment]
We discussed this and #142 at July 30, 2025 OMRSE meeting. Again, we'd like to revisit and reactivate the discussion.

**Comment by LCCarmody** [comment]
I no longer work on MAXO or for @pnrobinson, but I can chime in on the point of view. When specifically are the meetings?  It is possible @pnrobinson has assigned someone who may like to attend?

**Comment by pnrobinson** [comment]
@LCCarmody Not sure I understand the context? But happy to change with OMRSE if useful!

**Comment by hoganwr** [comment]
Discussed at monthly OMRSE meeting March 4, 2026. Need to go through one by one and see what already exists vs. what needs to be created. Will do that and update the tracker, creating NTRs for what does not exist yet and proposed modifications for what exists already.

**Comment by hoganwr** [comment]
We discussed this issue at today's OMRSE monthly meeting (2026-05-06). 

See related Google Sheet: https://docs.google.com/spreadsheets/d/1f2Rb3QdkYcCl2nkqqpiobxOLTd98v_ufhgdOCiNaYMo/edit?gid=1933693511#gid=1933693511 

We already have in OMRSE 'anesthesiologist role' and 'neurosurgeon role'. 

At a previous OMRSE meeting, Dr. Brochhausen graciously bequeathed to OMRSE ownership of all physician roles (at least) in his OOSTT ontology, which when we adopt those terms and manage them in OMRSE, get us another 5-8 terms. See the Sheet.

Then, we will develop classes for the remainder. 

The plan is to list out and discuss the new ones here, and when the discussions for various terms reach a stage of maturity, spin out NTR issues to create them from this discussion.

The discussion so far includes that MAXO's definition of rheumatologist evaluation leaves out treatment of systemic diseases treated by rheumatologists such as scleroderma, Sjogren's disease, and SLE.
